ui/desktop/app/routes/scopes/scope/projects/targets.js

```diff
@@ -52,6 +52,14 @@ export default class ScopesScopeProjectsTargetsRoute extends Route {
     // Recursively query all targets within the current scope
     let targets = await this.store.query('target', queryOptions);
 
+    // Load all targets in order to populate host_sources field.
+    // This is meant to be a temporary solution.
+    await Promise.all(
+      targets.map(async ({ id: target_id }) => {
+        await this.store.findRecord('target', target_id);
```

Comment:
You don't need this inner await if you're using `Promise.all`

Reply:
oops, thx ill fix that!